src/test/unit/lib/requests/thirdpartyRequests.test.js

```diff
@@ -0,0 +1,92 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Crosslake
+ - Lewis Daly <lewisd@crosslaketech.com>
+
+ --------------
+ ******/
+
+
+const fs = require('fs');
+jest.mock('http');
+jest.mock('stream');
+const http = require('http');
+
+const ThirdpartyRequests = require('../../../../lib/requests/thirdPartyRequests.js');
+const WSO2Auth = require('../../../../lib/WSO2Auth');
+
+const jwsSigningKey = fs.readFileSync(__dirname + '/../../data/jwsSigningKey.pem');
+
+
+describe.skip('ThirdpartyRequests', () => {
+    describe('postAuthorizations', () => {
+        const wso2Auth = new WSO2Auth({ logger: { log: () => { } }});
+        const config = {
+            // Disable logging in tests
+            // logger: { log: () => { } },
```

Comment:
if logger mocking function allows the special argument `keepQuiet` than you will not have to keep in source code commented code.

Reply:
proposition of mockLogger: https://github.com/eoln/sdk-scheme-adapter/blob/pisp/feat/305_authorizations/src/test/unit/mockLogger.js and how it could be used: https://github.com/eoln/sdk-scheme-adapter/blob/pisp/feat/305_authorizations/src/test/unit/inboundApi/handlers.test.js#L49